src/gramps.js

```diff
@@ -24,6 +24,19 @@ const getDefaultApolloOptions = options => ({
   ...options,
 });
 
+const checkTypeDefs = ({ schema, typeDefs, namespace }) => {
+  if (typeof schema === 'string') {
+    console.warn(
+      namespace,
+      'Type definitions must be exported as "typeDefs".',
+      'Use of "schema" has been deprecated and will be removed in a future release',
+    );
+    return typeDefs || schema;
+  } else {
+    return typeDefs;
+  }
+};
```

Comment:
Would it work to do this:
```js
const getTypeDefs = ({ schema, typeDefs, namespace }) => {
  if (typeof schema === 'string') {
    console.warn(
      namespace,
      'Type definitions must be exported as "typeDefs".',
      'Use of "schema" has been deprecated and will be removed in a future release',
    );
  }

  return typeDefs || schema;
};
```

This gives us the same result, I believe, and ensures we get a return.